c/include/libsbp/file_io.h

```diff
@@ -142,8 +142,8 @@ typedef struct SBP_ATTR_PACKED {
 typedef struct SBP_ATTR_PACKED {
   u32 sequence;    /**< Write sequence number */
   u32 offset;      /**< Offset into the file at which to start writing in bytes [bytes] */
-  char filename[0]; /**< Name of the file to write to */
```

Comment:
FWIW we could completely drop the data field from this struct, it's not used in our code... and I don't think it'd possible to use it?  Why does GCC even allow this to compile?